ui/src/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+interface FooterButtons {
+  label: string;
+  path: string;
+}
+
+const footerButtons: FooterButtons[] = [
+  { label: "Q&A", path: "/QA" },
```

Comment:
Talked to pillow, The path should be `/Q-and-A`

Reply:
Talked to pillow is the label? I can fix the path!

Reply:
Talked to pillow in terms of Pillow is her name, the vtuber character. Just put the path as the new one and leave the label alone